Exception when restore PermissionHelper in other Activity.

SDK Version:

0.9.10

Situation is like under:

1. Start `Main Activity`.
2. Start `Sub Activity`.
3. Generating report and send. (Attached `PermissionHelper` to `Sub Activity`)
4. Finish `Sub Activity`.
5. Exception.

Exception Log:

```
java.lang.RuntimeException: Unable to start activity ComponentInfo{com.my.main/com.my.main.activity.MainActivityUniversal}: android.app.Fragment$InstantiationException: Unable to instantiate fragment com.buglife.sdk.PermissionHelper: make sure class name exists, is public, and has an empty constructor that is public
  at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2416)
  at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2476)
  at android.app.ActivityThread.-wrap11(ActivityThread.java)
  at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1344)
  at android.os.Handler.dispatchMessage(Handler.java:102)
  at android.os.Looper.loop(Looper.java:148)
  at android.app.ActivityThread.main(ActivityThread.java:5422)
  at java.lang.reflect.Method.invoke(Native Method)
  at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:726)
  at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:616)
Caused by: android.app.Fragment$InstantiationException: Unable to instantiate fragment com.buglife.sdk.PermissionHelper: make sure class name exists, is public, and has an empty constructor that is public
  at android.app.Fragment.instantiate(Fragment.java:628)
  at android.app.FragmentState.instantiate(Fragment.java:106)
  at android.app.FragmentManagerImpl.restoreAllState(FragmentManager.java:1858)
  at android.app.FragmentController.restoreAllState(FragmentController.java:122)
  at android.app.Activity.onCreate(Activity.java:918)
  at com.my.main.activity.MainActivity.onCreate(MainActivity.kt:58)
  at com.my.main.activity.MainActivityUniversal.onCreate(MainActivityUniversal.kt:53)
  at android.app.Activity.performCreate(Activity.java:6251)
  at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1107)
  at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2369)
  at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2476) 
  at android.app.ActivityThread.-wrap11(ActivityThread.java) 
  at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1344) 
  at android.os.Handler.dispatchMessage(Handler.java:102) 
  at android.os.Looper.loop(Looper.java:148) 
  at android.app.ActivityThread.main(ActivityThread.java:5422) 
  at java.lang.reflect.Method.invoke(Native Method) 
  at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:726) 
  at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:616) 
Caused by: java.lang.IllegalAccessException: java.lang.Class<com.buglife.sdk.PermissionHelper> is not accessible from java.lang.Class<android.app.Fragment>
  at java.lang.Class.newInstance(Native Method)
  at android.app.Fragment.instantiate(Fragment.java:613)
  at android.app.FragmentState.instantiate(Fragment.java:106) 
  at android.app.FragmentManagerImpl.restoreAllState(FragmentManager.java:1858) 
  at android.app.FragmentController.restoreAllState(FragmentController.java:122) 
  at android.app.Activity.onCreate(Activity.java:918) 
  at com.my.main.activity.MainActivity.onCreate(MainActivity.kt:58) 
  at com.my.main.activity.MainActivityUniversal.onCreate(MainActivityUniversal.kt:53) 
  at android.app.Activity.performCreate(Activity.java:6251) 
  at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1107) 
  at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2369) 
  at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2476) 
  at android.app.ActivityThread.-wrap11(ActivityThread.java) 
  at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1344) 
  at android.os.Handler.dispatchMessage(Handler.java:102) 
  at android.os.Looper.loop(Looper.java:148) 
  at android.app.ActivityThread.main(ActivityThread.java:5422) 
  at java.lang.reflect.Method.invoke(Native Method) 
  at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:726) 
  at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:616) 
```

**Proposed solution:**

Please add `public` access modifiers in `PermissionHelper` class declare.

--- a/src/main/java/com/buglife/sdk/PermissionHelper.java
+++ b/src/main/java/com/buglife/sdk/PermissionHelper.java
@@ -28,7 +28,7 @@ import android.support.annotation.NonNull;
 import android.support.annotation.Nullable;
 import android.app.Fragment;
 
-final class PermissionHelper extends Fragment {
+public final class PermissionHelper extends Fragment {
     private static final int REQUEST_PERMISSIONS = 10;
     public static final String TAG = "com.buglife.PermissionHelper";
 

